cmd/csidriver/main.go

```diff
@@ -130,6 +151,8 @@ func init() {
 	rootCmd.Flags().AddGoFlagSet(flag.CommandLine)
 
 	rootCmd.Flags().StringVar(&cfgFilePath, "config", "/var/run/configmaps/config/config.yaml", "configuration file path")
+	rootCmd.Flags().StringVar(&shCfgFilePath, "shConfig", "/var/run/configmaps/config/sharedconfigmap-list.yaml", "Shared config configuration file path")
+	rootCmd.Flags().StringVar(&shSecretCfgFilePath, "shSecret", "/var/run/configmaps/config/sharedsecret-list.yaml", "Shared secret configuration file path")
```

Comment:
These need to be removed, and the `config` line above them needs to be updated to be a sliceVar.

Reply:
Done